africanlii/templates/africanlii/layouts/document_detail.html

```diff
@@ -27,9 +27,27 @@ <h1 class="heading-underlined mb-4">{{ document.title }}</h1>
               <div class="col-md-6">
                 {% block document-metadata-content %}
                   <dl class="row">
-                    {% if document.authoring_body %}
+                    {% if document.author %}
                       <dt class="col-4">Author</dt>
-                      <dd class="col-8 text-muted">{{ document.authoring_body }}</dd>
+                      <dd class="col-8 text-muted">{{ document.author }}</dd>
+                    {% endif %}
+
+                    {% if document.jurisdiction %}
+                      <dt class="col-4">Jurisdiction</dt>
+                      <dd class="col-8 text-muted">{{ document.jurisdiction }}</dd>
+                    {% endif %}
+
+                    {% if document.locality %}
+                      <dt class="col-4">Locality</dt>
+                      <dd class="col-8 text-muted">{{ document.locality }}</dd>
+                    {% endif %}
```

Comment:
Please format this like it was suggested in the issue: `Country · Locality` which is what we do in Indigo. That separator is a middot character: ·

Do this under the "Jurisdiction" heading.

We do this because "locality" is confusing to people.